iml-agent/src/action_plugins/stratagem/server.rs

```diff
@@ -253,6 +253,22 @@ pub fn generate_cooked_config(path: String, rd: Option<u64>, pd: Option<u64>) ->
         });
     }
 
+    if true {
+        let name = "filesync";
+
+        conf.device.groups.push(name.into());
+
+        conf.groups.push(StratagemGroup {
+            name: name.into(),
+            rules: vec![StratagemRule {
+                action: "LAT_SHELL_CMD_FID".into(),
+                expression: format!("&& != type S_IFDIR >= size 1048576"),
+                argument: "filesync".into(),
+                counter_name: Some("filesync".into()),
+            }],
+        });
+    }
+
```

Comment:
Should this be here? or should `generate_cooked_config()` take more arguments?

Reply:
This should get fixed, or the config should only be generated in one place, rather than here and in stratagem.py